src/map/lua/lua_baseentity.cpp

```diff
@@ -13878,6 +13949,30 @@ inline int32 CLuaBaseEntity::useJobAbility(lua_State* L)
     return 0;
 }
 
+inline int32 CLuaBaseEntity::useWeaponSkill(lua_State* L)
+{
+    DSP_DEBUG_BREAK_IF(m_PBaseEntity == nullptr);
+    DSP_DEBUG_BREAK_IF(!lua_isnumber(L, 1));
+    //DSP_DEBUG_BREAK_IF(lua_isnil(L, 2) || lua_isuserdata(L, 2));
+
+    CBattleEntity* target = nullptr;
+    if (lua_isnil(L, 2))
+        target = (CBattleEntity*)m_PBaseEntity;
+    else
+    {
```

Comment:
brace or don't brace but for goodness sake don't brace one branch of a conditional and not the other
```cpp
if things
{
    doShit();
}
else
{
    doOtherShit();
}
```
or
```cpp
if things
    doShit();
else
    doOtherShit();
```